src/modules/flow/http-client/http-client.c

```diff
@@ -60,6 +62,21 @@ struct http_client_node_type {
     int (*process_data)(struct sol_flow_node *node, struct sol_http_response *response);
 };
 
+static int
+machine_id_header_add(struct sol_http_param *params, char *id)
```

Comment:
while at that, also fix the `sol_platform_get_serial_number()` to be the same. These should all be consistent!